We assigned mutation rates to all possible non-synonymous variants using data from the Pancancer Analysis of Whole Genomes database (J. N. Weinstein, E. A. Collisson, G. B. Mills, K. R. Shaw, B. A. Ozenberger, K. Ellrott, I. Shmulevich, C. Sander, and J. M. Stuart. The cancer genome atlas pan-cancer analysis project. Nature genetics, 45(10):1113–1120, 2013). We queried a total of $28717344$ whole genome single point somatic mutations to construct a trinucleotide mutational signature, that is, the mutation rates specified by the identity of the bases that immediately flank the mutated base. We assigned mutation rates to all possible beneficial amino acid changes summing up the rates for each of its associated nucleotide changes. Such rates can be found in mutSignature_p53.txt.

Nominal_p53.txt provides the selection coefficients and mutation rates to construct the nominal distribution of Fig 4. *

Multi_p53.txt provides the mutation type, selection coefficients and observed counts to construct the nominal and fixed distributions for the amino acid changes associated to multi and single point mutations in Figure 5. *

* Original dara from Giacomelli, A.O., Yang, X., Lintner, R.E. et al. Mutational processes shape the landscape of TP53 mutations in human cancer. Nat Genet 50, 1381–1387 (2018). 
